Edits to new Decoding worker text added with PR 4266 (#4324)
EnterpriseDB/docs: product_docs/docs/pgd/5/reference/pgd-settings.mdx

### `bdr.enable_wal_decoder`

Enables logical change record (LCR) sending on a single node with a [decoding worker](../nodes#decoding-worker). By default, this setting is false. When set to true, a decoding worker process starts, and WAL senders send the LCRs it produces. If set back to false, any WAL senders using LCR are restarted and use the WAL directly.

!!! Note
    You also need to enable this setting on all nodes in the PGD group and set the `enable_wal_decoder` option to true on the group.

### `bdr.receive_lcr`

> Enables the node to request to use LCRs when subscribed to another node. By default, this setting is false. When this setting is true on a downstream node, the node requests that upstream nodes use LCRs when sending to it. If you set `bdr.enable_wal_decoder` to true on a node, also set this setting to `true`.

Reviewer: ```suggestion
When subscribing to another node, this setting enables the node to request the use of LCRs (logical change records) for the subscription. By default, this setting is false. When this setting is true on a downstream node, the node requests that upstream nodes use LCRs when sending to it. If you set `bdr.enable_wal_decoder` to true on a node, also set this setting to `true`.
```

!!! Note 

>     You also need to enable this setting on all nodes in the PGD group and set the `enable_wal_decoder` option to to true on the group.

Reviewer: ```suggestion
 You also need to enable this setting on all nodes in the PGD group and set the `enable_wal_decoder` option to to true on the group.
!!!
```

Fence rather than indent admonitions - it avoids triggering an admonition bug and is much more clearly defined.

### `bdr.lcr_cleanup_interval`

> LCR file cleanup interval. When the [decoding worker](../nodes#decoding-worker) is enabled, the decoding worker stores LCR files as a buffer. These files are periodically cleaned, and this setting controls the interval between any two consecutive cleanups. The default is 3 minutes. Setting it to zero disables cleanup.

Reviewer: This is a reference section and the reader is not reading the content sequentially. Keeping the full name and acronym (the explain the variable name) is reasonable. Run with the acronym first if needs be, but reference sections need to be relatively self-contained.

```suggestion
LCR (logical change record) file cleanup interval. When the [decoding worker](../nodes#decoding-worker) is enabled, the decoding worker stores LCR files as a buffer. These files are periodically cleaned, and this setting controls the interval between any two consecutive cleanups. The default is 3 minutes. Setting it to zero disables cleanup.
```

Author: Okay, but I'm going to switch them because the definition goes first and the initialism goes second.
